### PR TITLE
feat(medusa): Display admin url on start

### PR DESCRIPTION
**Pull request by adrien2p**
**What**
Display the admin url on server start in the console
Screenshot 2024-10-17 at 16 07 10

### CODE REVIEW

File: packages/medusa/src/commands/start.ts [hunk 1]
```diff
@@ -109,6 +114,15 @@ async function start(args: {
       const server = GracefulShutdownServer.create(
         http_.listen(port).on("listening", () => {
           logger.success(serverActivity, `Server is ready on port: ${port}`)
+
+          const {
+            admin: { path: adminPath },
```
Comment (shahednasser) [inline packages/medusa/src/commands/start.ts]: thought: shouldn't we check if the admin is disabled in the config?

**Review by shahednasser — APPROVED** [review]
Love it!

File: packages/medusa/src/commands/start.ts (hunk 1 above)
Comment (kasperkristensen) [reply packages/medusa/src/commands/start.ts]: Yes I think so to, so we don't print a link that goes to no where.

Also think we can skip printing this in production, as the actual link won't be http://localhost:9000/app in that case, and people aren't going to need a link to the admin there anyways.

**Review by kasperkristensen — APPROVED** [review]
LGTM! Just small nit on checking if admin is disabled, and not printing in production

File: packages/medusa/src/commands/start.ts (hunk 1 above)
Comment (adrien2p) [reply packages/medusa/src/commands/start.ts]: yes thats true 👍

Comment (adrien2p) [reply packages/medusa/src/commands/start.ts]: I have performed the changes, let me know if that's ok for you

Comment (kasperkristensen) [reply packages/medusa/src/commands/start.ts]: Looks good @adrien2p!

**Pull request merged**